src/utils/common.test.ts

```diff
@@ -0,0 +1,13 @@
+import { stripArrayTypeIfPresent } from './common';
+
+describe('CommonUtils', () => {
+  describe('stripArrayTypeIfPresent', () => {
+    it('remove array brackets from the type if present', () => {
```

Comment:
Nit: It looks like `stripArrayTypeIfPresent` will ignore a case like `string []`, where there is a space before `[]`. Perhaps we can add a test case for that as well?

Reply:
fixed

Reply:
I think it was correct before. You've now updated it to allow `Type []`, which is unspecified behaviour (the spec only references `Type` or `Type[]`). I didn't mean to suggest you update the regex, just that there was a test case missing

From https://eips.ethereum.org/EIPS/eip-712:
>Arrays are either fixed size or dynamic and denoted by Type[n] or Type[] respectively

Reply:
I updated the code and also changed regex to handle fixed size array like `string[5]`.

Reply:
Oh nice, never thought about fixed-sized arrays, and they were right in the quote. Good catch